packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/example.html

```diff
@@ -66,6 +66,26 @@ <h2>Examples</h2>
       <span slot="help">This is a help message</span>
     </gux-form-field-color>
   </fieldset>
+
+  <fieldset>
+    <legend>Label Info</legend>
+
+    <gux-form-field-color>
```

Comment:
This should be the same as the other examples

Reply:
Think these should be the same now